internal/pkg/cli/task_run.go

```diff
@@ -231,7 +233,7 @@ func (o *runTaskOpts) Validate() error {
 	}
 
 	if o.image != "" && o.dockerfilePath != "" {
-		return errors.New("cannot specify both image and Dockerfile path")
+		return errors.New("specified both image and Dockerfile path")
```

Comment:
nit: can we make this error format consistent like the ones in `validateRunnerTypeOptions`? or vice-versa

Reply:
Done! I had changed it but i forgot to push...